accessibility-in-action/platform
# feat: list and manage engagement participants

- [x] Allow admins and community connectors to view participant lists
- [x] Allow community connectors to invite participants up to maximum participant count
- [x] Allow participants to accept or decline invitations

### Testing Notes

1. Run `composer install` to get updated dependencies.
2. Run `php artisan migrate:fresh --seeder DevSeeder`
3. Find the project called "BlueSky Airlines Accessibility Project" and its engagement set to recruit via a community connector.
4. Attach an individual community connector to that engagement.
5. Log in as that community connector and visit the engagement page. You should see a call to action link at the top of the page which lets you access the "Manage Participants" section.
6. Try inviting an individual participant via their email address. See what happens when that participant declines the invitation or accepts the invitation.

### Prerequisites

If this PR changes PHP code or dependencies:

- [x] I've run `composer format` and fixed any code formatting issues.
- [x] I've run `composer analyze` and addressed any static analysis issues.
- [x] I've run `php artisan test` and ensured that all tests pass.
- [x] I've run `composer localize` to update localization source files and committed any changes.

If this PR changes CSS or JavaScript code or dependencies:

- [x] I've run `npm run lint` and fixed any linting issues.
- [x] I've run `npm run build` and ensured that CSS and JavaScript assets can be compiled.
